src/applications/coronavirus-chatbot/components/ChatbotLoadError.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export default function ChatbotLoadError(_props) {
+  return (
+    <AlertBox
+      headline="Error Loading Chatbot"
+      content="We are sorry there was an issue loading the chatbot, please refresh the page to try again. If the problem persists there may be a caching issue, please clear your cache and refresh the page."
```

Comment:
Do we need or did we get content review on this message? Just curious

Reply:
I dont know I am still new to the VA's way of doing things. @mchelen-gov mind weighing in here?

Reply:
@DanielleThierryUSDSVA do you have input on this?

Reply:
i think 'clear your cache' is not something most people would know how to do. Maybe pointing to a page with such instructions would be helpful as it's different for various browsers (e.g. triggering a force refresh)

Reply:
see https://github.com/department-of-veterans-affairs/vets-website/pull/12561#issuecomment-625428860 for updated text

Reply:
see https://github.com/department-of-veterans-affairs/vets-website/pull/12561#issuecomment-625428860 for updated text